umbra-js/src/classes/Umbra.ts

```diff
@@ -8,7 +8,6 @@ import {
   BigNumber,
   BigNumberish,
```

Comment:
*Arbitrary comment location*

I think we should just remove the "Copy payment link" function from the page completely. Not sure it translates well to a batch send context. If we want to consider adding it back later, we can create an issue for that, but I don't think it should be required to ship.

Reply:
Agreed, removed relevant lines